card/mod/05_standard/set/all/rich_html/editing.rb

```diff
@@ -19,7 +19,13 @@ def default_new_args args
 
     args[:optional_help] ||= :show
 
-    # name field / title
+    default_new_args_for_name_field_or_title args
+    default_new_args_for_type_field args
+    default_new_args_buttons args
+  end
+
+  def default_new_args_for_name_field_or_title args
```

Comment:
Assignment Branch Condition size for default_new_args_for_name_field_or_title is too high. [25.98/15]